Content.Shared/Eye/Blinding/SharedBlindingSystem.cs

```diff
@@ -34,7 +35,22 @@ private void OnUnequipped(EntityUid uid, BlindfoldComponent component, GotUnequi
             component.IsActive = false;
             if (!TryComp<BlindableComponent>(args.Equipee, out var blindComp))
                 return;
-            blindComp.Sources--;
+            AdjustBlindSources(args.Equipee, false, blindComp);
+        }
+
+        [PublicAPI]
+        public void AdjustBlindSources(EntityUid uid, bool Add, BlindableComponent? blindable = null)
+        {
+            if (!Resolve(uid, ref blindable, false))
+                return;
+
+            if (Add)
+            {
+                blindable.Sources++;
+            } else
+            {
+                blindable.Sources--;
+            }
```

Comment:
Are sources netsynced? How does this work with prediction?

Reply:
It's a networked component and I haven't noticed anything with the fakelag.